rocketmq-console/src/main/java/org/apache/rocketmq/console/support/AclClientRPCHookFactory.java

```diff
@@ -0,0 +1,51 @@
+
+package org.apache.rocketmq.console.support;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.apache.rocketmq.acl.common.AclClientRPCHook;
+import org.apache.rocketmq.acl.common.SessionCredentials;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Acl Helper.
+ *
+ * @author 莫那·鲁道
+ * @date 2019-04-18-09:27
+ */
+public class AclClientRPCHookFactory {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AclClientRPCHookFactory.class);
+
+    private ConcurrentHashMap<String, AclClientRPCHook> cache = new ConcurrentHashMap<>();
+
+    private AclClientRPCHookFactory(){}
+
+    public static AclClientRPCHookFactory getInstance() {
+        return AclClientRPCHookFactoryLazyHolder.INSTANCE;
+    }
+
+    private static class AclClientRPCHookFactoryLazyHolder {
+
+        private static final AclClientRPCHookFactory INSTANCE = new AclClientRPCHookFactory();
+    }
+
+
+    public AclClientRPCHook createAclClientRPCHook(String accessKey, String secretKey) {
+
+        if (accessKey != null && secretKey != null) {
```

Comment:
Here,I think it's nonvaluable if the accessKey or secretKey is empty String.So you can judge whether they is empty or not?